app/Http/Controllers/Kata7/StringEndController.php

```diff
@@ -4,23 +4,45 @@
 
 use App\Http\Controllers\Controller;
 use App\Http\Requests\Kata7\StringEndRequest;
+use App\Interfaces\CrudRepositoryInterface;
 use App\Service\Kata7\StringEndService;
+use Illuminate\Contracts\View\View;
 use Illuminate\Http\RedirectResponse;
 use Illuminate\Routing\Redirector;
+use Resources\views;
+
 
 class StringEndController extends Controller
 {
-    /**
-     * @var StringEndService
-     */
     protected StringEndService $stringEndService;
+    protected CrudRepositoryInterface $stringEndRepository;
 
     /**
      * @param StringEndService $stringEndService
+     * @param CrudRepositoryInterface $stringEndRepository
      */
-    public function __construct(StringEndService $stringEndService)
+    public function __construct(StringEndService $stringEndService, CrudRepositoryInterface $stringEndRepository)
```

Comment:
I really like that youre using an interface here.

Reply:
:)